### ISSUE TITLE
Doc review: Deploy VCHs

### ISSUE THREAD

**Issue by emlin** [vmware/vic#3340]
Review: https://github.com/vmware/vic/blob/master/doc/user_doc/vic_installation/deploy_vch.md

**Comment by emlin** [comment]
VCH does not register extension for any deployed VCH now. Instead, it provides options --ops-user, --ops-password to set operation user account for the new vch. Then customer is able set RBAC for that user in vsphere, for security reason.

**Comment by mhagen-vmware** [comment]
lgtm

**Comment by stuclem** [comment]
Thanks @emlin. This helps a lot with https://github.com/vmware/vic/issues/3298, https://github.com/vmware/vic/issues/2517, and https://github.com/vmware/vic/issues/1692 which are all still in the queue.

**Comment by hickeng** [comment]
Use the full term for the title: _Deploying Virtual Container Hosts_

> You install vSphere Integrated Containers Engine by deploying vSphere Integrated Containers Engine virtual container hosts (VCHs). You use the vic-machine utility to deploy a VCH.

This is mangled and confusing - I think we shouldn't be talking about installing VICE as it's meaningless. We can talk about installing the VICE UI plugin, and about deploying VCHs.

The vic-machine utility can deploy a VCH to the following:
* vCenter cluster
* standalone ESXi host (managed by vCenter)
* standalone ESXi host

_Perhaps add a link to the supported configurations here_

> The VCH allows you to use an ESXi host or vCenter Server instance as the Docker endpoint for a Docker client. The containers that Docker developers pull or create by using a Docker client are stored and managed in the vSphere environment.\

This is repetition of the overview and intro concepts -- I don't think it adds anything here and "managed in the vSphere environment" could be a very misleading term. I think we should expand on how container elements present in vSphere, and what management is expected to be done by devs via the Docker API vs that expected to be done by the viadmins.

> When you deploy a VCH, vic-machine registers the VCH as a vSphere extension. Authentication between the VCH and vSphere is handled via key pair authentication against the vSphere extension.

This is no longer the case - username/password is used, either the admin credentials supplied for/with --target or the -ops-user if present. We _should_ be able to take an SSO token in the password field, but that needs testing and confirming.


**NOTE: I'm unsure what value this page adds once changes are made - it's basically a secondary table of contents. Is it possible to make this page more of a task-based walk-through of a simple install, resulting in one of the basic examples. My thought is that gives people some of the core vic-machine configuration concepts before they get into the detailed breakdown**

**Comment by stuclem** [comment]
Thanks @emlin and @hickeng. Yes, this topic is pretty useless. Wrote it very early on and then pretty much didn't look at it again. It was rendered even more redundant by a recent content reorg that I did prior to RC. Everything here is said better (hopefully) elsewhere. Its primary / only real function is to group the more interesting VCH deployment topics together. It's a wrapper topic.

Deleted everything and left only "You use the vSphere Integrated Containers Engine `vic-machine` utility to deploy virtual container hosts" and the chapter TOC.

Closing as no further review required.
